salt/states/pkgbuild.py

```diff
@@ -237,7 +237,7 @@ def built(name,
     return ret
 
 
-def repo(name, keyid=None, env=None):
+def repo(name, keyid=None, env=None, use_passphrase=False, gnupghome='/etc/salt/gpgkeys', runas='builder'):
```

Comment:
Where does this `builder` user come from? Is this specific to your build system?